.github/workflows/tests.yml

```diff
@@ -18,10 +18,20 @@ jobs:
         with:
           dafny-version: "3.4.2"
 
-      - uses: actions/setup-python@v1
+      - name: Install Dafny report generator
+        run: dotnet tool install --global dafny-reportgenerator
 
       - name: Install lit 
         run: pip install lit OutputCheck
 
-      - name: Verify/Test Dafny code
-        run: lit --time-tests -v .
+      - name: Verify Dafny Code
+        run: lit --time-tests -v --param 'dafny_params=/verificationLogger:trx /verificationLogger:csv' .
+
+      - name: Generate Report
+        run: find . -name '*.csv' | xargs -t dafny-reportgenerator summarize-csv-results --max-duration-seconds 10
+
+      - uses: actions/upload-artifact@v2  # upload test results
+        if: success() || failure()        # run this step even if previous step failed
+        with:
+          name: verification-results
+          path: 'TestResults/*.trx'
```

Comment:
The build complains that these files aren't found. I think that's because the current logic for report generation chooses the last report type specified, rather than doing all of them. That should probably change.

Reply:
Actually no, Dafny definitely supports both enabling both loggers at once! The issue here is that the path pattern is wrong: initially I was trying to replace `lit` with just one or two top-level calls to `dafny`, in which case you ended up with the results in one top-level `TestResults` folder. But now that I'm leaving `lit` in place, you end up with a `TestResults` folder in every subdirectory instead. :)

I've fixed the pattern and added an option to fail if no files match. Nice catch!

Reply:
Also notice that unfortunately, the `workflows` trigger won't actually fire until this is merged, which makes it harder to verify. :(

Reply:
Ah, got it!